File: Encuentro #01/C#/Ejercicio_01/Ejercicio_01/Ponente.cs
using System;

namespace Ejercicio_01
{
    public class Ponente : Participante
    {
        public string Titulo_Ponencias { get; set; }
        public string Tematicas { get; set; }
        
        public Ponente() {}
        public Ponente(string nombre, string apellido, string email, string tituloPonencias, string tematicas)
            : base(nombre, apellido, email)
        {
            Titulo_Ponencias = tituloPonencias;
            Tematicas = tematicas;
        }
        
        public void LeerPonente()
        {
            LeerParticipante();
            Console.WriteLine("Ingrese el titulo de la ponencia");
            Titulo_Ponencias = Console.ReadLine();
            Console.WriteLine("Ingrese la tematica de la ponencia");
            Tematicas = Console.ReadLine();
        }
        
        public void mostrarPonente()
        {
            MostrarParticipante();
            Console.WriteLine($"Titulo: {Titulo_Ponencias}\nTematica: {Tematicas}");
        }
    }
}

File: Encuentro #01/Java/Ejercicio 01/src/com/pplam/modelos/Ponente.java
package com.pplam.modelos;

import java.util.Scanner;

public class Ponente extends Participante {
    private String titulo, tematica;

    public Ponente() {
        super();
    }

    /**
     * @param nombre nombre
     * @param apellido apellido
     * @param email email
     * @param titulo titulo
     * @param tematica tematica
     */
    public Ponente(String nombre, String apellido, String email, String titulo, String tematica) {
        super(nombre, apellido, email);
        this.setTitulo(titulo);
        this.setTematica(tematica);
    }

    public void leerPonente() {
        super.leerParticipante();
        System.out.println("Titulo: ");
        this.setTitulo(new Scanner(System.in).nextLine());

        System.out.println("Tematica: ");
        this.setTematica(new Scanner(System.in).nextLine());
    }

    public void mostrarPonente() {
        super.mostrarParticipante();
        System.out.println("\nTitulo:" + this.getTitulo() + "\nTematica: " + this.getTematica());
    }

    public String getTitulo() {
        return titulo;
    }

    /**
     * @param titulo titulo
     */
    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public String getTematica() {
        return tematica;
    }

    /**
     * @param tematica tematica
     */
    public void setTematica(String tematica) {
        this.tematica = tematica;
    }
}
